packages/evm/src/eof/errors.ts

```diff
@@ -0,0 +1,254 @@
+export enum EOFError {
+  // Stream Reader
+  OutOfBounds = 'Trying to read out of bounds',
+  VerifyUint = 'Uint does not match expected value ',
+  VerifyBytes = 'Bytes do not match expected value',
+
+  // Section Markers
+  FORMAT = 'err: invalid format',
+  MAGIC = 'err: invalid magic',
+  VERSION = `err: invalid eof version`,
+  KIND_TYPE = `err: expected kind types`,
+  KIND_CODE = `err: expected kind code`,
+  KIND_DATA = `err: expected kind data`,
+  TERMINATOR = `err: expected terminator`,
+
+  // Section Sizes
+  TypeSize = `missing type size`,
+  InvalidTypeSize = `err: type section size invalid`,
+  CodeSize = `missing code size`,
+  CodeSectionSize = `code section should be at least one byte`,
+  InvalidCodeSize = `code size does not match type size`,
+  DataSize = `missing data size`,
+  ContainerSize = 'missing container size',
+  ContainerSectionSize = 'container section should at least contain one section and at most 255 sections',
+
+  // Type Section
+  TypeSections = `err: mismatch of code sections count and type signatures`,
+  Inputs = 'expected inputs',
+  Outputs = 'expected outputs',
+  MaxInputs = 'inputs exceeds 127, the maximum, got: ',
+  MaxOutputs = 'outputs exceeds 127, the maximum, got: ',
+  Code0Inputs = 'first code section should have 0 inputs',
+  Code0Outputs = 'first code section should have 0x80 (terminating section) outputs',
+  MaxStackHeight = `expected maxStackHeight`,
+  MaxStackHeightLimit = `stack height limit of 1024 exceeded: `,
+
+  // Code/Data Section
+  MinCodeSections = `should have at least 1 code section`,
+  MaxCodeSections = `can have at most 1024 code sections`,
+  CodeSection = `expected a code section`,
+  DataSection = `Expected data section`,
+
+  // Container section
+  ContainerSection = 'expected a container section',
+  ContainerSectionMin = 'container section should be at least 1 byte',
+  InvalidEOFCreateTarget = 'EOFCREATE targets an undefined container',
+  InvalidRETURNContractTarget = 'RETURNCONTRACT targets an undefined container',
```

Comment:
This makes more sense if we make the full RETURNCONTRACT opcode name capitalized.

```suggestion
  InvalidRETURNCONTRACTTarget = 'RETURNCONTRACT targets an undefined container',
```